### PR TITLE
Added game package

### PR DESCRIPTION
**Pull request by Tcmiller12**
Added go file to contain function that runs game.

### CODE REVIEW

File: game/game.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+func (g Guessing) Play(){
+	g.trialcount = 1
+	source := rand.NewSource(time.Now().UnixNano())
+	randomizer := rand.New(source)
+	secrectNumber := randomizer.Intn(10)
+	fmt.Println("Guess the number")
+	for {
+		fmt.Println("Please input your guess")
+		fmt.Scan(&g.guess)
+		if g.trialcount > 3 {
```
Comment (eopeter) [inline game/game.go:22]: Make the Constant 3 an instance variable called AllowedCount

**Review by eopeter — CHANGES_REQUESTED** [review]

File: game/game.go (hunk 1 above)
Comment (Tcmiller12) [reply game/game.go]: Added  a new variable named "NumberOfTriesAllowed" that allows dev to adjust attempt count from main.go.

Comment (eopeter) [reply game/game.go]: Can we call it `AttemptsAllowed`

Comment (Tcmiller12) [reply game/game.go]: Changes have been made the variable's now called AttemptsAllowed

**Review by eopeter — APPROVED** [review]

**Pull request merged**